CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
Thanks to a silly hack from demo time, recommendations are being generated
in utility model pipeline.. I'm moving the code over to recommendation
pipeline however.

Do you remember our discussion a while ago on implementing the user model?
The issue was that alternatives were not attached to their original trips
in any way, and so the model could learn crazy things. What was the final
solution?


On Wednesday, April 29, 2015, shankari <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/recommendation_pipeline.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29336257>
> :
>
> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
> >          return []
> >
> >      def get_selected_user_utility_model(self, user_id):
> > -        return []
> > +        return UserUtilityModel.find_from_db(user_id)
> >
> >      def recommend_trips(self, trip_id, utility_model):
> >          return []
>
> @sdsingh <https://github.com/sdsingh>, where are recommendations being
> generated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29336257>.
>